osu.Server.Queues.ScorePump/Queue/ImportHighScoresCommand.cs

```diff
@@ -37,20 +37,39 @@ public class ImportHighScoresCommand : BaseCommand
         public int RulesetId { get; set; }
 
         /// <summary>
-        /// The high score ID to start the import process from. This can be used to resume an existing job, or perform catch-up on new scores.
+        /// The high score ID to start the import process from. This can be used to perform batch reimporting for special cases.
         /// </summary>
         [Option(CommandOptionType.SingleValue)]
-        public ulong StartId { get; set; }
+        public ulong? StartId { get; set; }
 
         /// <summary>
-        /// Whether existing legacy score IDs should be skipped rather than fail via an error. Defaults enabled.
+        /// Whether to adjust processing rate based on slave latency. Defaults to <c>false</c>.
         /// </summary>
-        [Option(CommandOptionType.SingleValue, Template = "--skip")]
+        [Option(CommandOptionType.SingleValue, Template = "--check-slave-latency")]
+        public bool CheckSlaveLatency { get; set; }
+
+        /// <summary>
+        /// Whether existing legacy score IDs should be skipped rather than fail via an error. Defaults to <c>true</c>.
+        /// </summary>
+        [Option(CommandOptionType.SingleValue, Template = "--skip-existing")]
         public bool SkipExisting { get; set; } = true;
 
+        /// <summary>
+        /// Whether to skip pushing imported score to the elasticsearch indexing queue.
+        /// </summary>
+        [Option(CommandOptionType.SingleValue, Template = "--skip-indexing")]
+        public bool SkipIndexing { get; set; }
+
+        /// <summary>
+        /// Whether to exit when there are no scores left at the tail end of the import. Defaults to <c>false</c>.
+        /// </summary>
+        public bool ExitOnCompletion { get; set; }
```

Comment:
Was this intended to have an `[Option]` on it?

Reply:
Yeah.